x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_header.scss

```diff
@@ -0,0 +1,26 @@
+.appSearchResultHeader {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &Column {
+    display: flex;
+    flex-wrap: wrap;
+
+    &--left {
+      justify-content: flex-start;
+
+      & > div {
+        margin-right: 27px;
+      }
+    }
+
+    &--right {
+      justify-content: flex-end;
+
+      & > div + div {
```

Comment:
[TODO for me to investigate later and provide a more concrete alternative] I'm not a huge fan of using unspecific `div` selectors - could we potentially use `:last-child` or `:first-child` selectors instead to get the margin/spacings we want?

Reply:
Yes, or maybe there's a more specific class name I can use. I'll investigate.

Reply:
375ecef